cfg/types.go

```diff
@@ -25,11 +32,22 @@ type (
 		OlderThan           string `mapstructure:"older-than"`
 		OrphanDeletionRegex string `mapstructure:"deletion-pattern"`
 	}
+	// LogConfig configures the log
 	LogConfig struct {
 		LogLevel string
 		Batch    bool
 		Verbose  bool
 	}
+	// ConfigMapConfig configures the config map
+	ConfigMapConfig struct {
```

Comment:
I'd suggest we use the same struct for both ConfigMap and Secret, since they're the same (e.g. `ResourceConfig`)

Reply:
Suggestion accepted